### PR TITLE
feat: release v0.1.0

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Hi @membphis we need to release a new version on this. I tried to do like `api7-lua-resty-http` but I'm not sure if I follow the correct process to release a forked repo.

### CODE REVIEW

**Review by membphis — APPROVED** [review]

File: rockspec/lua-tinyyaml-0.1-0.rockspec [hunk 1]
```diff
@@ -1,15 +1,15 @@
-package = "lua-tinyyaml"
-version = "0.1-0"
+package = "api7-lua-tinyyaml"
```
Comment (membphis) [inline rockspec/lua-tinyyaml-0.1-0.rockspec:1]: we can use this name `lua-tinyyaml`, if we need to release a new version, we can release a new version, eg: v2.0

File: rockspec/lua-tinyyaml-0.1-0.rockspec [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+   tag = "v0.1.0"
 }
 
 description = {
    summary = "a tiny yaml (subset) parser for pure lua",
-   homepage = "https://github.com/iresty/lua-tinyyaml",
+   homepage = "https://github.com/api7/lua-tinyyaml",
    license = "MIT License",
-   maintainer = "peposso"
+   maintainer = "Yuansheng Wang <membphis@gmail.com>"
```
Comment (membphis) [inline rockspec/lua-tinyyaml-0.1-0.rockspec:12]: we should keep the original maintainer name

**Review by membphis — CHANGES_REQUESTED** [review]

File: rockspec/lua-tinyyaml-0.1-0.rockspec (hunk 2 above)
Comment (membphis) [inline rockspec/lua-tinyyaml-0.1-0.rockspec:12]: please remove `maintainer ` field

**Review by membphis — CHANGES_REQUESTED** [review]

File: rockspec/lua-tinyyaml-0.1-0.rockspec (hunk 2 above)
Comment (Yiyiyimu) [reply rockspec/lua-tinyyaml-0.1-0.rockspec]: done

**Review by membphis — APPROVED** [review]

**Pull request merged**